### DISCUSSION TITLE
How to navigate to the parent directory of the current working directory?

### DISCUSSION THREAD

**Discussion by hopezh** [nvim-neo-tree/neo-tree.nvim#1331] [Q&A] [answered]
E.g, I'm working in this directory: `~/AppData/Local/oatVim4` in `nvim`, as shown below. 

How to navigate to the **parent directory** of the **current working directory**, i.e. `~/AppData/Local`, using `neo-tree`? 

![image](https://github.com/nvim-neo-tree/neo-tree.nvim/assets/23435810/f2fddd5d-4c50-4aa5-9aea-b401d540c4f4)

**Comment by cseickel** [comment, marked as answer]
Backspace
